src/visitor.rs

```diff
@@ -190,6 +179,7 @@ impl<'a, 'v> visit::Visitor<'v> for FmtVisitor<'a> {
             ast::Item_::ItemUse(ref vp) => {
                 self.format_import(item.vis, vp, item.span);
             }
+            // TODO(#78): format traits and impl definitions.
```

Comment:
nit: here and below, can we use FIXME rather than TODO for long-term things to be fixed?